### ISSUE TITLE
Particles Stuck Between Arrows and Wormholes Continue Forever

### ISSUE THREAD

**Issue by Justintime50** [alexanderson1993/reaction#1]
If the map is setup where you have two wormholes and two arrows (tutorial level 9) and you don't land the particles onto an atom, they have the potential to continue going back and forth forever. There are two problems with this setup:

1. It's distracting to the user while they do the rest of the level
2. It's possibly resource intensive to keep them going?

A good way to fix this may be to have a check that says on the third entry of a particle to a wormhole, if it hasn't landed yet on an atom and stopped, simply destroy the particle.

![Screen Shot 2021-08-19 at 10 53 51 AM](https://user-images.githubusercontent.com/39606064/130111198-a4e77176-7f19-4e0b-b5ed-558e7fa245be.png)

**Comment by alexanderson1993** [comment]
I'll grant you that it might be distracting. However, I can assure you it uses a negligible amount of resources. I'll still keep your suggestion in mind.

**Comment by Justintime50** [comment]
I spent a bunch of time trying to figure out how to do this and determined the game isn't currently setup in a way to provide enough state per particle to justify overhauling it to allow this simple feature that really provides no value. I tried using the `setParticle()` function; however, that's really meant to set the location only and incrementing a number of "bounces" would be a side-effect. Additionally, incrementing a value and attaching it to each particle seemed a little cumbersome - I'll close this. Feel free to point me in a direction if I overlooked a simple way this could be implemented.
